### PR TITLE
Add crate: Polars

### PR DESCRIPTION
**Pull request by ritchie46**
Polars is a blazingly fast in memory DataFrame library. I believe it is the most feature complete option in the Rust landscape.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -979,6 +979,7 @@ See also [About Rust’s Machine Learning Community](https://medium.com/@autumn_
 
 * [bluss/ndarray](https://github.com/rust-ndarray/ndarray) — N-dimensional array with array views, multidimensional slicing, and efficient operations
 * [kernelmachine/utah](https://github.com/kernelmachine/utah) — Dataframe structure and operations in Rust
+* [ritchie46/polars/](https://github.com/ritchie46/polars) - Fast feature complete DataFrame library
```
Comment (palfrey) [inline README.md:982]: Please add the Travis build badge and lose the "/" from the end of the name bit. Thanks!

Comment (ritchie46) [reply README.md]: Done!

File: README.md [hunk 2]
```diff
@@ -979,6 +979,7 @@ See also [About Rust’s Machine Learning Community](https://medium.com/@autumn_
 
 * [bluss/ndarray](https://github.com/rust-ndarray/ndarray) — N-dimensional array with array views, multidimensional slicing, and efficient operations
 * [kernelmachine/utah](https://github.com/kernelmachine/utah) — Dataframe structure and operations in Rust
+* [ritchie46/polars](https://github.com/ritchie46/polars) - Fast feature complete DataFrame library [![Build Status](https://travis-ci.com/ritchie46/polars.svg?branch=master)](https://travis-ci.com/ritchie46/polars)
```
Comment (palfrey) [inline README.md:982]: The build badge URL 301's to https://api.travis-ci.com/ritchie46/polars.svg?branch=master which is what's breaking the build. Can you fix that please?

Comment (ritchie46) [reply README.md]: Right! Fixed.

**Pull request merged**